homeassistant/components/roborock/manifest.json

```diff
@@ -8,6 +8,7 @@
   "loggers": ["roborock"],
   "requirements": [
     "python-roborock==0.36.2",
-    "vacuum-map-parser-roborock==0.1.1"
+    "vacuum-map-parser-roborock==0.1.1",
+    "dacite==1.8.1"
```

Comment:
I think we should set a minimum version in `package_constraints.txt` instead. Add a line to `gen_requirements_all.py` with a comment explaining it and then run the script.

We don't want to pin to a specific version as that's what the custom integration did wrong in the first place. Then we'd have to bump that version all the time when a new version is released to solve conflicts.

Reply:
Thanks - changed!